src/Button/Button.tsx

```diff
@@ -0,0 +1,75 @@
+import { ButtonHTMLAttributes, FC, forwardRef, ReactNode } from 'react';
+import { BackgroundColorProps } from 'styled-system';
+
+import ButtonContent from './ButtonContent';
+import ButtonSpinner from './ButtonSpinner';
+import { Size } from './sizes';
+import StyledButton from './StyledButton';
+import Text from '../Text';
+import { Variant } from './variants';
+import { Intent } from '../theme/colors/intent';
+import { PaletteColor } from '../theme/colors/palette';
+
+type Props = ButtonHTMLAttributes<HTMLButtonElement> &
+  BackgroundColorProps & {
+    fullWidth?: boolean | boolean[];
+    iconStart: ReactNode;
```

Comment:
`iconStart` and `iconEnd` were optional in the original. Should it still be that way?

Reply:
Yeah, they should still be optional -- good catch, thanks.